docs/configuration/configuration-file.mdx

```diff
@@ -5,224 +5,465 @@ import { StyledContent } from '../../components/styled.elements';
+| `{apiName}` |  | `patternProperties` | REQUIRED. Identifies an API definition file. |
+| `{apiName}@{apiVersion}` |  | `patternProperties` | Identifies multiple versions of an API definition file. `{apiName}` is required but `@version` is optional. |
+|  | `root` | `string` | REQUIRED. The relative path to where a definition file is stored. |
+|  | `labels` | `array` | Assigns Redocly Workflows organization-wide labels to a definition file. |
+|  | `lint` | `object` | Defines linting rules at the file level (overrides global `lint` configuration). |
+|  | `features.openapi` | `object` | Configures theming and functionality for an API definition. |
 
-registry:
-# The API registry options
 
-# future products may be configured by extending this file
+### Examples
+
+```yaml
+apis:
+  main:
+    root: ./openapi.yaml
+    labels:
+      - main
+    lint: []
+    features.openapi: []
+  production:
+    root: ./production.yaml
+    labels:
+      - production
+    lint: []
+    features.openapi: []
+  sandbox:
+    root: ./sandbox.yaml
+    labels:
+      - sandbox
+    lint: {}
+    features.openapi: {}
 ```
 
+Given the configuration above and the following command, `./production.yaml` is validated because it's the only one currently using the name `production`:
 
-### organization
+```shell
+openapi lint production
+```
 
-The `organization` property requires the organization ID of your Redocly Workflows organization.
-When specified in the configuration file, it can be used by the `openapi push` command.
-The `push` command uses the following order of precedence: first, it takes the organization ID from command-line arguments (if provided).
-If the organization ID is not provided explicitly, it takes it from the configuration file.
+If you run `lint` without specifying a name, all API definitions listed in the `apis` section will get validated:
 
-Note that the organization ID is not the same as the organization name in Workflows.
+```shell
+openapi lint
+```
 
-To find the organization ID, log into Workflows and access the **API registry** page.
-In your browser's address bar, find the URL of the page.
-The segment after `app.redocly.com/org/` is your organization ID.
-For example, if the URL is `app.redocly.com/org/test_docs`, the organization ID is `test_docs`.
+When you run the [preview-docs command](../commands/preview-docs.md), you *must* specify a file. As a shortcut, you can reference definitions listed in the `apis` section.
+
+```shell
+openapi preview-docs production
+```
+
+### Optional properties supported by the `apis` section
+The `apis` section supports several optional properties for each definition file listed.
 
+#### The `labels` property
+Use `labels` to assign one or more existing Redocly Workflows organization-wide labels to your OpenAPI definitions. You must have a Redocly Workflows account with an active organization, and the top-level `organization` property must be added to `redocly.yaml`.
 
-### apis
+```yaml
```

Comment:
suggestion: it could be helpful to show the `organization` property in this example, too